images/miq-app/Dockerfile

```diff
@@ -76,7 +76,7 @@ RUN mkdir -p ${APP_ROOT} && \
 
 ## Add ManageIQ source from local directory (dockerfile development) or from Github (official build)
 #ADD . ${APP_ROOT}
-RUN curl -L https://github.com/ManageIQ/manageiq/tarball/${REF} | tar vxz -C ${APP_ROOT} --strip 1
+RUN curl -L https://github.com/carbonin/manageiq/tarball/use_ansible_service_in_containers | tar vxz -C ${APP_ROOT} --strip 1
```

Comment:
This will go away once https://github.com/ManageIQ/manageiq/pull/15423 is merged.